### ISSUE TITLE
org.glassfish.grizzly:grizzly-http-client missing from BOM

### ISSUE THREAD

**Issue by vandmo** [eclipse-ee4j/glassfish-grizzly#2048]
As the title says.

Maybe the BOM should be in its own repository and have a separate lifecycle?
Since fixing this would require a 2.4.4.1 version of the BOM.

#2032 is missing as well.

**Comment by vandmo** [comment]
Another minor issue is that the BOM contains grizzly-compression that does not actually exist.

**Comment by smillidge** [comment]
Do you want to raise a PR to tidy it up?

**Comment by vandmo** [comment]
There is one PR already for the connection-pool missing. #2032
Removing grizzly-compression I can create a PR for but the first one hasn't been merged for 2 months.

IIRC the problem was that I got the wrong version of grizzly-http-client because some grizzly artifact had an too old dependency on that library. I did not dig deeper into that at that time though, I just added the latest to my own dependentManagement section.

Since the grizzly-http-client artifact has its own lifecycle, https://github.com/eclipse-ee4j/grizzly-ahc, I think that the grizzly-bom needs that as well and I am not sure how to create a PR for creating a new project.
